Database/Corrections/cataQuestFixes.lua

```diff
@@ -3005,6 +3005,9 @@ function CataQuestFixes.Load()
         [25468] = { -- Forces of Nature: Faerie Dragons
             [questKeys.objectives] = {nil,nil,nil,nil,{{{5276,5278},5276,"Faerie Dragons Rallied",Questie.ICON_TYPE_INTERACT}}},
         },
+        [25469] = { -- Forces of Nature: Mountain Giants
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_MOUNT_UP,l10n("Mountain Giants Rallied"),0,{{"monster",40026}}}},
```

Comment:
`objective` was correct already. But not of type "killCredit" (5th index) but of type "monster" (1st index):

```suggestion
            [questKeys.objectives] = {{{40026}}},
```

Reply:
https://github.com/Questie/Questie/pull/6255/commits/eaccca163f9fc1cd1a0ed728367bd2abb5991552